app/checkout/[userId]/page.tsx

```diff
@@ -56,6 +57,24 @@ export default function CheckOut({ params }: CheckOutProps) {
         router.push(`/`);
     };
 
+    const onChange = (visitData: Visit) => {
+        setVisitFormData(visitData);
+    };
```

Comment:
Nit: we can pass setVisitFormData into VisitInfoForm's onChange prop

Reply:
Yes that would more clean. Thanks for the sugesstion!